src/api.js

```diff
@@ -1,6 +1,12 @@
+import { employees } from './demo'
 const apiPath = '/hr-server/api/v1/';
 
+const isDemo = false //Change to true for demo data
```

Comment:
Could you please make sure the semicolon usage is consistent throughout this file?

Reply:
Good point, I'm not used to adding semicolons for js. Npm has a package called `Prettier` that goes through and formats the code nicely. I'll add it in to this project in the next PR.